### DISCUSSION TITLE
How to create a new workspace when scrolling with the mouse wheel?

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#6015] [Q&A]
It is really useful to scroll with my mouse wheel and cycle my workspaces.

However, is it possible to have it create a new empty workspace after reaching the last one?

So, if I have 3 workspaces and scroll, instead of going to workspace 1, it creates workspace 4.

**Comment by orestisfl** [comment]
See [5.5. Mouse button commands](https://i3wm.org/docs/userguide.html#_mouse_button_commands). Specifically:
```
# execute custom script when scrolling downwards
bindsym button5 exec ~/.i3/scripts/custom_wheel_down
```

Then, your script can do something similar to this: https://github.com/i3/i3/issues/5484#issuecomment-1508172220
